src/Components/AuthShared/ButtonForm/ButtonForm.tsx

```diff
@@ -1,11 +1,28 @@
-import React from "react";
 import TrueIcone from "../../../Icones/TrueIcone";
-
-export default function ButtonForm({ text }) {
+import { Loader } from "lucide-react";
+interface Iprops {
+  text: string;
+  isSubmitting: boolean;
+}
+export default function ButtonForm({ text, isSubmitting }: Iprops) {
   return (
-    <button className="w-[150px] h-[50px] bg-[#F5F5F5] font-bold text-lg rounded-lg text-[#000] flex justify-center items-center  gap-2">
-      <p>{text}</p>
-      <TrueIcone />
+    <button
+      className={`w-[150px] h-[50px] ${
+        isSubmitting ? "bg-[#D3D3D3]" : "bg-[#F5F5F5]"
```

Comment:
If these colors are repeated in the design, we can extend the colors in the tailwind extend in the tailwind config file and include it.